plugins/cleanupListOfValues.js

```diff
@@ -1,8 +1,45 @@
 import { removeLeadingZero } from '../lib/svgo/tools.js';
 
+/**
+ * @typedef {import('json-schema-typed').JSONSchema} JSONSchema
+ */
+
 export const name = 'cleanupListOfValues';
 export const description = 'rounds list of values to the fixed precision';
 
+/** @type {JSONSchema} */
+export const schema = {
+  type: 'object',
+  properties: {
+    floatPrecision: {
+      title: 'Remove',
+      description:
+        'Number of decimal places to round to, using conventional rounding rules.',
+      type: 'number',
+      default: 3,
+    },
```

Comment:
Since I'd want to represent this as a slider, what maximum value would you be happy with here?

Reply:
I'm 99% sure we have a limit of 20 somewhere for `floatPrecision`. However, I wanted some time to peek around in case it varies between plugins.

For now, I'll assume it'll be `minimum: 0, maximum: 20, multipleOf: 1`. But it's always possible that in this case or others the conclusion is that there shouldn't be a maximum.